paper_sections/04-examples.Rmd

```diff
@@ -313,10 +314,14 @@ plot_grid(pltG, pltGS,pltGI, ncol = 3, align = "hv", axis = "lrtb",labels=c("a",
 
 The AIC values indicate that both model *GS* (`r round(AIC(zoo_daph_modGS), 2)`) and *GI*
 (`r round(AIC(zoo_daph_modGI), 2)`) are better fits than model *G* (`r round(AIC(zoo_daph_modG), 2)`),
-but models *GI* fits somewhat better than model *GS* [^AIC_note].  There does not seem to be a large amount of inter-lake variability (the effective degrees of freedom per lake are low in models *GS* & *GI*).  Plots for all three models
+but model *GI* fits somewhat better than model *GS* [^AIC_note].  There does not seem to be a large amount of inter-lake variability (the effective degrees of freedom per lake are low in models *GS* & *GI*).  Plots for all three models
 (Fig. \ref{fig:daph_smooth}) show that Mendota, Monona, and Kegonsa lakes are very close to the average and to one another for both models, but Waubesa shows evidence of a more pronounced spring bloom and lower winter abundances. 
 
-[^AIC_note]: When comparing models via AIC, we use the standard rule of thumb from @burnham_model_1998, where models that differ by 2 units or less from the lowest AIC model have substantial support, and those differing by more than 4 units having less support. 
+[^AIC_note]: When comparing models via AIC, we use the standard rule of thumb from @burnham_model_1998, where models that differ by 2 units or less from the lowest AIC model have substantial support, and those differing by more than 4 units have less support. 
+
+<!-- It is extremely hard to see the more pronounced spring bloom in these plots.  I suggest
+plotting the in a single row to accentuate the vertical differences, and truncating the y-axis
+at a much lower value - 15? -->
```

Comment:
This is a good point; it is hard to make out the bloom. Really, it's being driven by one outlier in Waubesa. I've tried truncating the *y* axis at just above 20, and replaced the one outlier with a star (that I'd discuss in the caption, making it clear that the outlier was truncated for visual clarity). What do you think of this as a revised version for the figure? 
![capture](https://user-images.githubusercontent.com/1554249/52238837-8d7deb80-28a7-11e9-826e-53aec7acc4f3.PNG)